packages/pilot/src/pages/LoggedArea/dynamicImports.js

```diff
@@ -40,12 +40,17 @@ const PaymentLinks = lazy(() => import(
   /* webpackChunkName: "payment-links" */ '../PaymentLinks'
 ))
 
+const Settlements = lazy(() => import(
+  /* webpackChunkName: "payment-links" */ '../Settlements'
```

Comment:
```suggestion
  /* webpackChunkName: "settlements" */ '../Settlements'
```

essa notação define o nome do chunk